vars/provisionNodes.groovy

```diff
@@ -161,6 +161,7 @@ EOF'''
                                                   python-debuginfo             \
                                                   libcmocka python-pathlib     \
                                                   python2-numpy git            \
+                                                  clustershell                 \
```

Comment:
Do we actually need clustershell or will python2-clustershell suffice?  We are using the python API of it yes?

Reply:
It can work with python2-clustershell too. Let me change it to Python2.